fe1-web/components/ChirpCard.tsx

```diff
@@ -63,7 +105,7 @@ const ChirpCard = (props: IPropTypes) => {
 const propTypes = {
   sender: PropTypes.string.isRequired,
   text: PropTypes.string.isRequired,
-  time: PropTypes.instanceOf(Timestamp).isRequired,
+  time: PropTypes.number.isRequired,
```

Comment:
Why not directly taking a chirp object as property?

Reply:
That's a really good remark, thanks a lot for the suggestion. I didn't think about it but it is much more practical.